rules/standard_rules/impossible_travel_login.py

```diff
@@ -124,7 +124,11 @@ def rule(event):
         last_login = []
         for l_l in tmp_last_login:
             last_login.append(dumps(l_l))
-    last_login_stats = loads(last_login.pop())
+
+    if last_login:
+        last_login_stats = loads(last_login.pop())
+    else:
+        return False
```

Comment:
Returning here if there is no last login prevents a login from ever being saved

Reply:
We should either save the current log in info before returning or move this after that logic

Reply:
Instead of returning False, in the latest commit I attempted to set it a key from the current log event